React/wbooks/src/redux/allBooks/actions.js

```diff
@@ -0,0 +1,34 @@
+import UserService from '../../Services/UserService';
+
+export const allBooksFetchActions = {
+  FETCH_BEGIN: '@@ALL_BOOKS/FETCH_BEGIN',
+  FETCH_SUCCESS: '@@ALL_BOOKS/FETCH_SUCCESS',
+  FETCH_FAILURE: '@@ALL_BOOKS/FETCH_FAILURE'
+};
+
+export const allBooksFetchBegin = () => ({
```

Comment:
Put all these methods inside an ActionCreators object, as you did with auth reducer